src/main/java/org/alliancegenome/curation_api/model/entities/ontology/DOTerm.java

```diff
@@ -17,4 +25,34 @@
 @AGRCurationSchemaVersion(min = LinkMLSchemaConstants.MIN_ONTOLOGY_RELEASE, max = LinkMLSchemaConstants.MAX_ONTOLOGY_RELEASE, dependencies = { OntologyTerm.class })
 public class DOTerm extends OntologyTerm {
 
+	@ToString.Exclude
+	@OneToMany(mappedBy = "diseaseAnnotationObject")
+	private List<GeneDiseaseAnnotation> geneDiseaseAnnotations;
+	@ToString.Exclude
+	@OneToMany(mappedBy = "diseaseAnnotationObject")
+	private List<AlleleDiseaseAnnotation> alleleDiseaseAnnotations;
+	@ToString.Exclude
+	@OneToMany(mappedBy = "diseaseAnnotationObject")
+	private List<AGMDiseaseAnnotation> agmDiseaseAnnotations;
+
+	public  List<GeneDiseaseAnnotation> getGeneDiseaseAnnotations() {
```

Comment:
This overrides the default behavior of pulling all annotations, this logic needs to go into the builder and not be in the Object.

Reply:
Yes but it overrides the default behavior. Anyone calling these methods excepting to get all annotations will not. And there is not indication that there is a filter going on. Business logic shouldn't be in the model.

Reply:
Ok the change to method name works but how about something like getActiveGeneAnnotations?

Reply:
I could go with getPublic<xxx> method name...